docs/user/resources/04-50-gcp-vpc-peering.md

```diff
@@ -0,0 +1,56 @@
+# GcpVpcPeering Custom Resource
+
+The `gcpvpcpeering.cloud-resources.kyma-project.io` custom resource (CR) describes the VPC peering
+connection that you can create to allow communication between Kyma and a remote VPC in Google Cloud Platform (GCP).
+It allows you to consume services available on the remote VPC from the Kyma cluster.
+
+### Required Permissions on the Remote Project
```

Comment:
```suggestion
### Required Permissions on the Remote Project <!-- {docsify-ignore} -->
```
I noticed that you use `<!-- {docsify-ignore} -->` for all headings of the docs located in `/resources` and `/tutorials` folders, so let's add it also here